misc/config/config.yaml.nydus.tmpl

```diff
@@ -46,6 +46,8 @@ converter:
       builder: nydus-image
       # specify nydus format version, possible values: `5`, `6` (EROFS-compatible), default is `5`
       rafs_version: 5
+      # specify nydus compression algorithm, `lz4block` by default
```

Comment:
specify nydus blob compression algorithm, possible values: `none`, `lz4_block`, `gzip`, `zstd`, default is `lz4_block`?

Reply:
done.